aml_storage/block.py

```diff
@@ -1,33 +1,40 @@
 import gc
 import copy
 import ctypes
-from typing import Tuple, List
+from typing import List, Tuple, Generator
 
 from ._c_lib import _get_library
-from ._c_api import aml_label_kind, aml_labels_t, aml_array_t
+from ._c_api import aml_labels_t, aml_array_t
 
 from .status import _check_pointer
 from .labels import Labels
 
-from .data import AmlData, Array, aml_array_t_to_python_object
+from .data import AmlData, Array, aml_array_to_python_object
 
 
 class Block:
     """
-    Basic building block for descriptor. A single block contains a 3-dimensional
-    :py:class:`aml_storage.data.Array`, and three sets of :py:class:`Labels`
-    (one for each dimension).
+    Basic building block for descriptor. A single block contains a n-dimensional
```

Comment:
I not sure how strictly we would like to follow [PEP257](https://peps.python.org/pep-0257/#multi-line-docstrings) but I would stick with a first line short description and then a new paragraph.

```suggestion
    Basic building block for descriptor. 
    
    A single block contains a n-dimensional
```

Reply:
Thanks a lot for the review! 

I have an upcoming PR which renames a bunch of types/properties, I'll add the doc changes to this other PR.

Reply:
Totally fine with this. If you want, I can also look at the documentation afterwards? We could add links to numpy and torch with intersphinx.

Reply:
another set of eyes on the documentation would be great!